fda-model/metrics/SystemPerformanceResults.cs

```diff
@@ -230,7 +239,7 @@ public bool Equals(SystemPerformanceResults projectPerformanceResults)
             }
             return true;
         }
-        public AssuranceResultStorage GetAssurance(string type, double standardNonExceedanceProbabilityForAssuranceOfTargetOrLevee = 0)
+        internal AssuranceResultStorage GetAssurance(string type, double standardNonExceedanceProbabilityForAssuranceOfTargetOrLevee = 0)
```

Comment:
just to narrow down what options developers have to get results. I think leaving these other things public could kind of muddy up how someone should obtain the results. E.g. use MeanAEP() instead of GetAssurance(AEP).AssuranceHistogram.Mean